apps/web/core/components/pages/time-and-activity/export/time-activity-pdf.tsx

```
@@ -0,0 +1,228 @@
+		totalRecords: number;
+	};
+}
+
+export function TimeActivityPDF({ data, title, startDate, endDate, appliedFilters, summary }: TimeActivityPDFProps) {
+	// Data is already transformed, just use it directly
+	const transformedData = React.useMemo(() => {
+		console.log('🔍 PDF Component - Receiving transformed data:', {
+			dataLength: data?.length || 0,
+			sampleData: data?.[0],
+			isTransformed: data?.[0]?.member ? true : false
+		});
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)